steps/mentor-calibre-lvs/lvs.runset.template

```diff
@@ -4,10 +4,11 @@
 *lvsRunDir: ./
 *lvsLayoutPaths: inputs/design_merged.gds
 *lvsLayoutPrimary: ${design_name}
-*lvsSourcePath: inputs/design.lvs.v
+*lvsSourcePath: inputs/merged.v
 *lvsSourceSystem: VERILOG
 *lvsSourcePrimary: ${design_name}
 *lvsSpiceFile: lvs.extracted.sp
+*lvsRecognizeGates: NONE
```

Comment:
Is there any documentation we can put here in a README.md about this option?

Reply:
I just scanned the mentor documentation and I think the high-level summary is that this option prevents LVS from trying to infer higher-level structures (inverters, AOI, etc.) to do logical pin swapping. There was probably some structure in our GDS that didn't fit their standard pattern so when they inferred a gate from it something unexpected was connected to the power/ground pins.